web/src/ApiButton.tsx

```diff
@@ -293,14 +293,14 @@ function ApiButtonWithOptions(props: ApiButtonWithOptionsProps & ButtonProps) {
       >
         {props.submit}
         <ApiButtonInputsToggleButton
+          {...buttonProps}
```

Comment:
the `aria-label` that gets set on the parent component is unintentionally passed down to this `ToggleButton` in `buttonProps`, so moving the props up here allows the correct label to be specified on this component.